readme.md

```diff
@@ -53,98 +50,64 @@ Broken into a variety of functional areas, the eligibility criteria is a checkli
-
--The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
+- The project operates within moral and ethical standards of the .NET Foundation.
+- The project is aligned with the philosophy and guidelines for collaborative development.
+- The project is built on the .NET platform and/or creates value within the .NET ecosystem.
+- The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
 
 **Code**
 
--The project produces software for distribution to the public at no charge.
-
--The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
-
--The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
-
--When applicable, project must use reproducible build settings in its toolchain.
-
--The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
-
--The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
-
--The project code signs their artifacts as appropriate.
+- The project produces software for distribution to the public at no charge.
+- The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
+- The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
 
 **Licenses and Copyright**
 
--The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
-
--Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
-
--Committers are bound by a Contributor License Agreement.
-
--The copyright ownership of everything that the project produces is clearly defined and documented.
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
```

Comment:
Where can people find a list of licenses approved by the .NET Foundation? We should have a link to the exact list if we plan to curate allowed permissive licenses.

Reply:
I agree with list of licensed but disagree with granting this much power to the OSI.

See https://github.com/dotnet-foundation/projects/pull/54#discussion_r382249773 for discussion.